website/docs/sidebar.md

```diff
@@ -227,3 +227,18 @@ module.exports = {
   },
 };
 ```
+
+### Collapsible categories
+
+For sites with a sizable amount of content, we support the option to expand/collapse the links and subcategories under categories. Categories are collapsible by default, but so that they are always expanded, set `themeConfig.sidebarCollapsible` to `false`:
```

Comment:
@yangshun feel free to change the text, I may not have written it so well. After that I will add corresponding changes to other versions of docs.